Source/SOS2Compat/SOS2Compat/Harmony/Harmony_ShipInteriorMoveShipNullGuards.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using HarmonyLib;
+using SaveOurShip2;
+using Verse;
+
+namespace CombatExtended.Compatibility.SOS2Compat;
+
+// Guards ShipInteriorMod2.MoveShip predicates against null Things produced by downstream mods.
```

Comment:
Should this be in SOS2 proper? Or is this something that SOS2 + CE together makes more likely?